test/libs/datasets/custom_aggregations_test.py

```diff
@@ -1,22 +1,22 @@
-import pytest
-
 from covidactnow.datapublic.common_fields import CommonFields
 
 from libs import pipeline
 from libs.datasets import combined_datasets
 from libs.datasets import custom_aggregations
 
 
-@pytest.mark.skip(
-    reason="Disabled due to failure. https://trello.com/c/7YidIL1H/752-fix-testreplacedccounty-test"
-)
 def test_replace_dc_county(nyc_region):
     dc_state_region = pipeline.Region.from_fips("11")
     dc_county_region = pipeline.Region.from_fips("11001")
 
     dataset = combined_datasets.load_us_timeseries_dataset().get_regions_subset(
         [nyc_region, dc_state_region, dc_county_region]
     )
+    dc_state_rows = (
```

Comment:
Add `TODO(tom): Find some way to have this test read data that hasn't already gone through replace_dc_county_with_state_data and remove this hack.`